docs/configuration.md

```diff
@@ -10,6 +10,12 @@ For more information on how the configuration is being loaded and used, please r
 ### Application Config
 Certain features of the React application import variables from an [AppConfig](https://github.com/lyft/amundsenfrontendlibrary/blob/master/amundsen_application/static/js/config/config.ts#L5) object. The configuration can be customized by modifying [config-custom.ts](https://github.com/lyft/amundsenfrontendlibrary/blob/master/amundsen_application/static/js/config/config-custom.ts).
 
+#### Example: Add a custom logo
+
+1. Add your logo to the folder in `amundsen_application/static/images/`
+2. Set the the `logoPath` key on the `AppConfigCustom` object in [config-custom.ts](https://github.com/lyft/amundsenfrontendlibrary/blob/master/amundsen_application/static/js/config/config-custom.ts). For example Lyft uses the value `"/static/images/lyft-logo.svg"`
+3. Rebuild/redeploy. The `npm run build` step will rebuild scripts file to reference the new logo image, and the `python3 setup.py install` step will updated deployed files and make the new image available on the server.
```

Comment:
@jornh to your note `...the tricky part really is the rebuilding steps which is common across AppConfig and Font/Styling.` Feel free to also take Step 3 from this example and add it either at the beginning or end of the whole "React Application" section, since as you've pointed out it's a common common step and we won't want to write this exact same step each time for every example we add. Fine to merge as is though until we add more examples. Thanks!

Reply:
Merge away!